gammapy/cube/models.py

```diff
@@ -188,8 +189,8 @@ class SkyModelMapEvaluator(object):
         Sky model
     exposure : `~gammapy.maps.Map`
         Exposure map
-    psf : TODO
-        PSF or PSF kernel
+    psf : `~gammapy.irf.EnergyDependentMultiGaussPSF`
+        PSF
```

Comment:
@registerrier  I decided to accept PSF objects (not kernels) here, ok? What other kinds of PSF objects might appear here (besides EnergyDependentMultiGaussPSF)

Reply:
Yes it seems simpler to accept a `PSFKernel`rather than the full (non-reduced) IRF. An `EnergyDependentTablePSF` might be a possibility too, but you would need to create a `PSFKernel` anyway.

Another option, for later might be to pass a `PSFMap` (once #1432 is merged) since this can produce the relevant `PSFKernel` if you have the position.

Reply:
EDIT: Only ``PSFKernels`` acceptec everywhere now (after offline discussion with @cdeil)